source/Calamari.Tests/KubernetesFixtures/HelmUpgradeFixture.cs

```diff
@@ -325,6 +325,18 @@ public void AdditionalArgumentsPassed()
             result.AssertOutputMatches("[helm|\\\\helm\"] upgrade (.*) --dry-run");
         }
 
+        [Test]
+        [RequiresNonFreeBSDPlatform]
+        [RequiresNon32BitWindows]
+        [RequiresNonMac]
+        [Category(TestCategory.PlatformAgnostic)]
+        public void WhenTheChartDirectoryVariableIsSet_TheChartAtThatLocationIsUsed()
```

Comment:
Would it be good to have some tests for the fallback behaviour. Variable is set, but didn't find it.

I've got a separate comment with my opinions on what we should do in that case, but either way would be good to have a test for it. Or are these really expensive?